client/src/components/userButton/userButton.sass

```diff
@@ -0,0 +1,81 @@
+.userLogo
+	height: 35px
+	width: 35px
+	margin: 5px
+	//margin-right: 5px
+	//margin-bottom: 5px
+
+.userLogo
+	&:hover
+		cursor: pointer
+
+.userBar
+	display: flex
+	flex-direction: row
+
+.userBar div
+	display: flex
+	justify-content: center
+	flex-direction: column
+
+.sideBarLogo, .floatingButtons
+	display: flex
+	justify-content: flex-end
+
+.userSideBar
+	border: 0
+	border-radius: 0.6rem
+	background-color: #D3D3D3
+	margin-right: 5px
+	box-shadow: 0 0 8px #a9a9a9
+
+.borderContainer
+	border-bottom: 1px solid gray
+	margin: 5px 10px
+
+.buttonContainer
+	margin: 5px 10px
+	border: 0
+	border-radius: 0.6rem
+	&:hover
+		cursor: pointer
+		background-color: #a9a9a9
```

Comment:
![image](https://user-images.githubusercontent.com/7780602/133536534-45d4035f-6a65-4d29-97be-7d43ac1abfc3.png)

This needs more work I think, the hover feels really off

Reply:
Are you okay with the hover when the user is logged in? We could move the log in to the right a little bit to match the indent on the logged in section.

![image](https://user-images.githubusercontent.com/61528501/134436419-3d2428dc-036f-4386-8706-34b7e3994a3c.png)

We could also add a bit of spacing on the top and bottom to keep a bit of extra background gray.

Reply:
Sorry, that's a really big picture.

Reply:
Okay, I've added a bit of space between the hover and the edge. Fixed the log out button as well at the same time.

Reply:
It having a hover effect is fine, if even encouraged. The actual behavior of the hover effect I think is what needs work. Can be a TODO for later though

Reply:
You're thinking the hover is less of the width? Or changes in some other way? Different colour/bolding/other

Reply:
Yeah, the hover being less than the whole width feels kinda awkward, plus how round it is. I wouldnt worry about it too much for now, more just noting it feels odd